spyder/app/cli_options.py

```diff
@@ -135,6 +135,13 @@ def get_options(argv=None):
         default=False,
         help="Start Spyder with a clean configuration directory"
     )
+    parser.add_argument(
+        '--offline',
+        dest="offline",
```

Comment:
```suggestion
        '--no-web-widgets',
        dest="no-web-widgets",
```

Spyder can work offline, so this is misleading. Instead, I suggest this change to make things more explicit.

Reply:
I agree, this is a better flag.